Client/Impl/Worker/JobWorkerBuilder.cs

```diff
@@ -64,6 +65,18 @@ public IJobWorkerBuilderStep3 Handler(AsyncJobHandler handler)
             return this;
         }
 
+        public IJobWorkerBuilderStep3 TenantIds(IList<string> tenantIds)
+        {
+            Request.TenantIds.Clear();
```

Comment:
🤔 Do we really want to clear the list?

This behaves different from the rest of the API where we can chain multiple calls. My expectation would be that it just adds together.

Example:

command.TenantIds(12, 34).TenantIds(56) -> results in {12, 34, 56) and not (56)

Reply:
Hi @Zelldon 

thanks for the review.
Yes that makes sense. I changed it so that it is aligned with the rest of the API.

Reply:
Thank you !